PubSub/src/ResourceNameTrait.php

```diff
@@ -123,6 +123,6 @@ private function isFullyQualifiedName($type, $name)
                 $type
             ));
         }
-        return (preg_match($this->regexes[$type], $name) === 1);
+        return (preg_match($this->regexes[$type], $name ?? '') === 1);
```

Comment:
We can't accept this because `??` is not valid on some versions of PHP which this library supports.
Try this instead:
```suggestion
        return (preg_match($this->regexes[$type], empty($name) ? '' : $name) === 1);
```

Reply:
Applied in 71ccaa68b1f27179a70f825fe4a572efb9cd4ac1. Thanks!